server/index.js

```diff
@@ -54,7 +57,16 @@ io.on('connection', (socket) => {
     // received signal to start a session from the host of a room, emit redirect signal to all guests and host
     socket.on('begin', ( user , callback) => {
         console.log('begin session signal received')
-        io.to(user.room).emit('sessionMembers', { room: user.room, users: getUsersInRoom(user.room), host: getHost(user.room)});
+
+        // get list of top 10 movies to send to all guests of the room
+        console.log('getting top 10')
+        getTop10().then((top10) => {
+            // TODO add call to redis to initialize swiping room here
+            io.to(user.room).emit('sessionMembers', { room: user.room, users: getUsersInRoom(user.room), host: getHost(user.room), top10: top10});
```

Comment:
did you also alter the socket.on function to be expecting this top10 parameter?

Reply:
yes I did, the socket.on function in lobby.js will expect this top10 parameter :)

Reply:
awesome LGTM!